ibm_catalog.json

```diff
@@ -3,7 +3,7 @@
       {
         "name": "deploy-arch-ibm-slz-mas",
         "label": "Maximo Application Suite",
-        "product_kind": "solution",
+        "Category": "Enterprise applications",
```

Comment:
`"product_kind": "solution"` is a required field here. Please add it back. If you want to add this category, you need to add `enterprise_app` to the `tags` array below

Reply:
It's added back..& added "Enterprise applications" under tags array..